doc/source/ray-core/handling-dependencies.rst

```diff
@@ -537,6 +537,13 @@ Currently, three types of remote URIs are supported for hosting ``working_dir``
 
     - ``runtime_env = {"working_dir": "gs://example_bucket/example_file.zip"}``
 
+- ``FILE``: ``FILE`` refers to URIs starting with ``file://`` that point to compressed packages stored on a local file system or a network file system.
+  To use packages via ``FILE`` URIs, you must have the ``smart_open`` library (you can install it using ``pip install smart_open``).
```

Comment:
Hm, I think we could probably use normal open for this case, just need to find the one line runtime env to special case.

Reply:
Good idea, I fixed it!